app/models/forms/address_form.rb

```diff
@@ -0,0 +1,29 @@
+class AddressForm
+  include ActiveModel::Model
+  include Virtus.model
+
+  attr_accessor :billing
```

Comment:
These form object is weird as hell! https://webuild.envato.com/blog/creating-form-objects-with-activemodel-and-virtus/ , http://hawkins.io/2014/01/form_objects_with_virtus/ check these out . Also you can remove validations from the corresponding models if there is a form (but not necessary all validations).